front/app/containers/EventsPage/EventsViewer/eventsViewerUtils.ts

```diff
@@ -0,0 +1,22 @@
+export function sliceEventsToPage(
+  events: number[],
+  currentPage: number,
+  eventsPerPage: number
+) {
+  const numberOfEvents = events.length;
+
+  if (numberOfEvents === 0) return [];
+  if (numberOfEvents <= 10) return events;
+
+  const startIndex = (currentPage - 1) * eventsPerPage;
+  const endIndex = startIndex + eventsPerPage;
+
+  return events.slice(startIndex, endIndex);
+}
+
+export function getNumberOfPages(
+  numberOfEvents: number,
+  eventsPerPage: number
+) {
+  return Math.ceil(numberOfEvents / eventsPerPage);
+}
```

Comment:
Is this used to paginate? The api is the best place to handle this, otherwise we're not solving the issue of very heavy response sizes, and if we have the logic in 2 places, then we need to maintain it in both FE and BE.